nuxt.config.js

```diff
@@ -53,10 +53,9 @@ export default {
     },
 
     postcss: {
-      plugins: [
-        require('tailwindcss'),
-        require('autoprefixer'),
-      ]
+      plugins: {
+        'tailwindcss: {}
```

Comment:
To pass a custom path, one could use the following syntax
```suggestion
        'tailwindcss: './tailwind.config.js' // or via path.join
```